### DISCUSSION TITLE
What happens if I use Optimize iPhone Storage ?

### DISCUSSION THREAD

**Discussion by KatyPurry** [immich-app/immich#134] [Q&A]
Hello,

What happens if I use the [Optimize Iphone Storage](https://support.apple.com/en-us/HT205703) option ? Will it download the high-resolution version before backup to Immish ?

Thank you !

**Comment by alextran1502** [comment]
Hmm, I am not sure about this. Would you mind checking the photos or videos that have been uploaded to the designated location if it has the same file size as your file on the phone?
